src/chapter09graphs/GraphsAdjacenyList.py

```diff
@@ -86,14 +86,15 @@ def setPrevious(self, current):
     def getPrevious(self, current):
         return self.previous
 	
-    def getEdges(self):
+    def get_edges(self):
         edges = []
-	for v in G:
```

Comment:
---> 47         for v in G:
     48             for w in v.get_connections():
     49                 vid = v.get_vertex_ID()

NameError: name 'G' is not defined